charts/payment-integration/templates/ingress.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+    app: {{ .Chart.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  annotations:
+{{ toYaml .Values.ingress.annotations | indent 4 }}
+spec:
+  backend:
+    serviceName: {{ .Chart.Name }}
```

Comment:
name of the service resource is `{{ template "payment.fullname" . }}`

Reply:
121f2ec